[2022-02-03 14:53:53,315] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: FHV_data_ingestion_gcs_dag.local_to_gcs_task scheduled__2020-06-01T00:00:00+00:00 [queued]>
[2022-02-03 14:53:53,363] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: FHV_data_ingestion_gcs_dag.local_to_gcs_task scheduled__2020-06-01T00:00:00+00:00 [queued]>
[2022-02-03 14:53:53,365] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 14:53:53,367] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-03 14:53:53,368] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 14:53:53,415] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2020-06-01 00:00:00+00:00
[2022-02-03 14:53:53,426] {standard_task_runner.py:52} INFO - Started process 1830 to run task
[2022-02-03 14:53:53,454] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'FHV_data_ingestion_gcs_dag', 'local_to_gcs_task', 'scheduled__2020-06-01T00:00:00+00:00', '--job-id', '485', '--raw', '--subdir', 'DAGS_FOLDER/FHV_injection_dag.py', '--cfg-path', '/tmp/tmpr9zc3er6', '--error-file', '/tmp/tmpj6ghrnqv']
[2022-02-03 14:53:53,458] {standard_task_runner.py:77} INFO - Job 485: Subtask local_to_gcs_task
[2022-02-03 14:53:53,627] {logging_mixin.py:109} INFO - Running <TaskInstance: FHV_data_ingestion_gcs_dag.local_to_gcs_task scheduled__2020-06-01T00:00:00+00:00 [running]> on host 700aeb16bae0
[2022-02-03 14:53:53,751] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-03 14:53:53,818] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=FHV_data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2020-06-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-06-01T00:00:00+00:00
[2022-02-03 14:53:55,936] {python.py:175} INFO - Done. Returned value was: None
[2022-02-03 14:53:55,970] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=FHV_data_ingestion_gcs_dag, task_id=local_to_gcs_task, execution_date=20200601T000000, start_date=20220203T145353, end_date=20220203T145355
[2022-02-03 14:53:56,075] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-03 14:53:56,171] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-03 15:15:48,632] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: FHV_data_ingestion_gcs_dag.local_to_gcs_task scheduled__2020-06-01T00:00:00+00:00 [queued]>
[2022-02-03 15:15:48,644] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: FHV_data_ingestion_gcs_dag.local_to_gcs_task scheduled__2020-06-01T00:00:00+00:00 [queued]>
[2022-02-03 15:15:48,644] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 15:15:48,645] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-03 15:15:48,646] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 15:15:48,657] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2020-06-01 00:00:00+00:00
[2022-02-03 15:15:48,661] {standard_task_runner.py:52} INFO - Started process 3266 to run task
[2022-02-03 15:15:48,667] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'FHV_data_ingestion_gcs_dag', 'local_to_gcs_task', 'scheduled__2020-06-01T00:00:00+00:00', '--job-id', '581', '--raw', '--subdir', 'DAGS_FOLDER/FHV_injection_dag.py', '--cfg-path', '/tmp/tmpdb3_2x9y', '--error-file', '/tmp/tmpju06z938']
[2022-02-03 15:15:48,669] {standard_task_runner.py:77} INFO - Job 581: Subtask local_to_gcs_task
[2022-02-03 15:15:48,738] {logging_mixin.py:109} INFO - Running <TaskInstance: FHV_data_ingestion_gcs_dag.local_to_gcs_task scheduled__2020-06-01T00:00:00+00:00 [running]> on host 700aeb16bae0
[2022-02-03 15:15:48,778] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-03 15:15:48,797] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=FHV_data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2020-06-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-06-01T00:00:00+00:00
[2022-02-03 15:15:50,944] {python.py:175} INFO - Done. Returned value was: None
[2022-02-03 15:15:50,963] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=FHV_data_ingestion_gcs_dag, task_id=local_to_gcs_task, execution_date=20200601T000000, start_date=20220203T151548, end_date=20220203T151550
[2022-02-03 15:15:51,013] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-03 15:15:51,048] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
